src/Core/SonarArtefactKeys.cs

```diff
@@ -67,6 +67,7 @@ public static class SonarRuleRepoKeys
         public const string Cpp = "cpp";
 
         public const string CSharpSecurityRules = "roslyn.sonaranalyzer.security.cs";
+        public const string CSharpHotspotSecurityRules = "csharpsquid";
```

Comment:
Is it right to call it "HotspotSecurity"? I believe this is just the normal csharp repository, no? (that the analyzer strips out the repository for the editor, and we should do the same for hotspots, so a better name would be simply CSharpRules)

Reply:
It is just the normal repo, so `CSharpRules` would do.

FYI the diagnostic descriptors for "normal" C# Roslyn rules just use the simple `Sxxx` values as the rule ID, so that is was is reported in the IDE i.e. there is nothing to strip out. The only reason we have a problem with security rules and hotspot rules is that we are fetching the data from the server, which uses the composite IDs.